### PR TITLE
Enable logging and add various values

### PR DESCRIPTION
**Pull request by prensing**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 1]
```diff
@@ -149,8 +149,10 @@ public static ShooterSpeeds calculateShooterSpeeds(double distance, boolean uppe
     // periodically update the values of motors for shooter to SmartDashboard
     @Override
     public void periodic() {
-        SmartDashboard.putNumber("shooter/left_rpm", getLeftRpm());
-        SmartDashboard.putNumber("shooter/right_rpm", getRightRpm());
+        SmartDashboard.putNumber("shooter/leftRpm", getLeftRpm());
+        SmartDashboard.putNumber("shooter/rightRpm", getRightRpm());
+        SmartDashboard.putNumber("shooter/leftCurrent", m_leftShooterMotor.getOutputCurrent());
+        SmartDashboard.putNumber("shooter/rightCurrent", m_rightShooterMotor.getOutputCurrent());
```
Comment (coachFitz0807) [inline src/main/java/frc/robot/subsystems/Shooter.java:155]: Do we want to also log the feeder speed and current to see if the current will tell us if we have the note?

File: src/main/java/frc/robot/swerve/DriveController.java [hunk 2]
```diff
@@ -7,4 +7,7 @@ public interface DriveController {
     public double getStateVelocity();
     // get wheel distance
     public double getWheelDistance();
+        
+    // update the smart dashboard
+    public void updateSmartDashboard(String sdPrefix);
 }
```
Comment (coachFitz0807) [inline src/main/java/frc/robot/swerve/DriveController.java:13]: As it turns out, the implementation of this method in both NeoDriveController and FalconDrive Controller are identical. THe differences are in the getStateVelocity methods. We could include a default implementation here and remove it from those classes to be overridden if needed in any future derived class.

**Review by Weepfrog — APPROVED** [review]
seems good!

File: src/main/java/frc/robot/swerve/DriveController.java (hunk 2 above)
Comment (prensing) [reply src/main/java/frc/robot/swerve/DriveController.java:13]: I figured it was better to let the real Drive controller do the monitoring, for future expansion. Also "DriveController" is an interface, so not sure we can add code.

**Pull request merged**